src/header/header.scss

```diff
@@ -161,45 +162,62 @@
     @include iui-header-button-icon;
   }
 
-  .iui-header-button {
+  .iui-header-button.iui-header-button { // boost specificity
```

Comment:
Why this change is needed?

Reply:
Because I moved the class out of `iui-left`, the specificity was lowered and was breaking styling, so I had to boost it back up

Reply:
Will this boost in specificity be needed after #363 passes?

Reply:
Hopefully not, but we'll see